mobility_pipeline/lib/overlap.py

```diff
@@ -0,0 +1,9 @@
+from shapely.geometry import Polygon
+
+def computeOverlap(enclosing, polygons):
+    overlapPercentages = []
+    for idx in range(len(polygons)):
```

Comment:
I think it's generally considered better style to use [`enumerate`](https://docs.python.org/3/library/functions.html?highlight=enumerate#enumerate) instead of the index-based for loop style common in Java/C++.